release-team/role-handbooks/enhancements/README.md

```diff
@@ -83,26 +83,22 @@ It is important that this process be followed and documentation remain up-to-dat
-- Examine the `Removed From Milestone` tab and visit those issues to see if they have been added to the current milestone. These enhancements likely slipped because of docs or testing. Ping owners to make sure they are targeted for the current milestone. When complete, clear the issues from the tab.
 - Find Issues from previous milestone that have graduated to Stable. Remove `tracked/yes` or `tracked/no` labels. Check to see if the KEP status has been updated to `implemented`. If it has, close the issue. If it has not, ask the issue contact to both update the KEP status field and close the Enhancement issue once the update PR has merged.
-- Clear out previous milestone issues. Ping owners to get them to drop milestone or add to the current milestone so it can be tracked. This can be a responsibility for shadow roles.
 - Close previous milestone by ensuring that there are no open issues/PRs in that milestone.
-- Issues in the current milestone should have the `tracked/yes` label applied and their status in the tracking sheet set to `Tracked`. Ensure they provide the required information and labels to be accepted into the milestone. If they do not meet the requirements, let the Issue owner know. set it's status within the tracking sheet to `At Risk` and add a comment describing which requirements are missing.
 - Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list with a call for enhancements. View prior examples for references ([1](https://groups.google.com/forum/#!msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ) [2](https://groups.google.com/forum/#!msg/kubernetes-dev/T-kIHtgS5J4/ZjKTsCf_AwAJ;context-place=msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ)).
 
 #### Pre-Enhancements Freeze (Week 1)
 
 - Gather Shadows to have them read this handbook, watch the [k/enhancements][k/enhancements] repo to get notified on all issues, and give expectations on what the process looks like and their particular role. If possible, try to schedule a call with the shadows to get them accustomed to the team. This helps as a great team building exercise.
-- Go to every open issue that is marked with `tracked/no` and ping the owner to see if the enhancement should be included in the current milestone. If there is no graduation or major change, leave as-is. If the enhancement is graduating or undergoing a major change, remove the `tracked/no` label and add `tracked/yes`, add the issue to the current milestone, and set it's status to `Tracked` within the tracking sheet if it meets the requirements. If it is missing some  of the requirements (e.g. KEP in a provisional state), let the issue owner know and add it as `At Risk`. Another great task for shadow help or some automation.
+- Ask each SIG leads to enter the KEPs that should be tracked for the release into the tracking sheet under `KEP Collection` tab.
```

Comment:
Two bullets down, we say "attend every SIG meeting over the span of 2-3 weeks". Should this become part of that effort (in addition to an email blast and possibly an update at the Chairs/TL meeting, just to ensure we have our bases covered? I think, especially since this is a change, we need to over communicate this?  Maybe call out the required communication points vs just "Ask each SIG lead"

Reply:
- I can modify the "attend every SIG meting..." to include the information that is required to share.
- I can modify the added "Ask each SIG lead" to specify the required communication (ex. email to k-dev and reaching out in slack #chairs-and-techleads and each sig channels) 

How does that sound? Also, am open to any other way to communicate kep collection. Not sure if there is any other tool we can use other than email/slack.

Reply:
Made changes to reflect the suggestion above. I kept it high level -- "communicate the opt-in process". Not sure. if you want a detailed explanation of how to opt-in? If that's the case, I like to first make sure that the current proposal to use a separate tab (KEP collection) in the tracking sheet for SIG leads to enter KEP info is the way we prefer.